From 829b0c29ca5d7864490251f9ce118f9ffaa53b45 Mon Sep 17 00:00:00 2001
From: George Zacharia <george.zcharia@gmail.com>
Date: Fri, 17 Jan 2020 10:21:27 +0530
Subject: [PATCH] Use a nicer pin icon

Isn't pin same as bookmark?
Use bookmark icon from material icon
Also use a different icon for unpin
---
 .../src/main/res/drawable/ic_unpin_black_24dp.xml        | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 presentation/src/main/res/drawable/ic_unpin_black_24dp.xml

diff --git a/presentation/src/main/res/drawable/ic_unpin_black_24dp.xml b/presentation/src/main/res/drawable/ic_unpin_black_24dp.xml
new file mode 100644
index 00000000..6ab9a0c6
--- /dev/null
+++ b/presentation/src/main/res/drawable/ic_unpin_black_24dp.xml
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
+        android:height="24dp"
+        android:viewportWidth="24.0"
+        android:viewportHeight="24.0">
+    <path
+        android:fillColor="#FF000000"
+        android:pathData="M17,3L7,3c-1.1,0 -1.99,0.9 -1.99,2L5,21l7,-3 7,3L19,5c0,-1.1 -0.9,-2 -2,-2zM17,18l-5,-2.18L7,18L7,5h10v13z"/>
+</vector>
